.github/workflows/update-snapshot.yml

```diff
@@ -0,0 +1,91 @@
+name: Update to new data snapshot (regenerate colors and sharing images).
+
+on:
```

Comment:
so this is still manually called right now right?

Reply:
Yeah.  My thinking is that we can add a scheduled workflow that just triggers it every morning.

And if we need to kick one off manually (e.g. the morning snapshot was bad), then we can do that too.